datafusion/src/logical_plan/expr.rs

```diff
@@ -1120,7 +1120,7 @@ pub fn columnize_expr(e: Expr, input_schema: &DFSchema) -> Expr {
 
 /// Recursively call [`Column::normalize`] on all Column expressions
 /// in the `expr` expression tree.
-pub fn normalize_col(e: Expr, schemas: &[&DFSchemaRef]) -> Result<Expr> {
+pub fn normalize_col(expr: Expr, schemas: &[&DFSchemaRef]) -> Result<Expr> {
```

Comment:
Changed to use the name `expr` to be more consistent with the rest of the codebase